.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea/
 
+
```

Comment:
@Finistere there's already a newline here, do we need this?

Reply:
nop, just a leftover, fixing it